config.json

```diff
@@ -192,7 +192,8 @@
     "loadingOverlay-backgroundColor": "#ffffff",
     "pace-progress-backgroundColor": "#989898",
     "spinner-borderColor-dark": "#989898",
-    "spinner-borderColor-light": "#ffffff"
+    "spinner-borderColor-light": "#ffffff",
+    "show_content_navigation": true
```

Comment:
The problem with this is that default values is `true` so when merchants get the update they won't have this setting in their config and therefore the pages will disappear.

What you should do instead is, name this `hide_content_navigation` and inverse the logic